packages/framework/src/Resources/views/Admin/Content/Superadmin/mailWhitelist.html.twig

```diff
@@ -6,21 +6,5 @@
 {% block main_content %}
     {{ render(controller('Shopsys\\FrameworkBundle\\Controller\\Admin\\DomainController::domainTabsAction')) }}
 
-    <div class="in-message in-message--info margin-bottom-30">
-        {{ 'E-mails are sent to addresses which comply with at least one regular expression. You can use <a href="https://regex101.com">https://regex101.com</a> for validating regular expressions.'|trans|raw }}
-    </div>
-
-    {% if isWhitelistForced %}
-        <div class="in-message in-message--warning margin-bottom-30">
-            {{ 'E-mail whitelist is currently forced by MAILER_FORCE_WHITELIST environment variable. Whitelist is always active.'|trans }}
-        </div>
-    {% endif %}
```

Comment:
We no longer need these informations?

Reply:
They're defined in the form type, so they're rendered at the more appropriate places (above significant field)